cmd/telemetrygen/internal/logs/worker.go

```diff
@@ -10,78 +10,90 @@ import (
 	"sync/atomic"
 	"time"
 
-	"go.opentelemetry.io/collector/pdata/pcommon"
-	"go.opentelemetry.io/collector/pdata/plog"
 	"go.opentelemetry.io/otel/attribute"
+	"go.opentelemetry.io/otel/log"
+	sdklog "go.opentelemetry.io/otel/sdk/log"
+	"go.opentelemetry.io/otel/sdk/log/logtest"
 	"go.opentelemetry.io/otel/sdk/resource"
+	"go.opentelemetry.io/otel/trace"
 	"go.uber.org/zap"
 	"golang.org/x/time/rate"
 )
 
 type worker struct {
-	running        *atomic.Bool        // pointer to shared flag that indicates it's time to stop the test
-	numLogs        int                 // how many logs the worker has to generate (only when duration==0)
-	body           string              // the body of the log
-	severityNumber plog.SeverityNumber // the severityNumber of the log
-	severityText   string              // the severityText of the log
-	totalDuration  time.Duration       // how long to run the test for (overrides `numLogs`)
-	limitPerSecond rate.Limit          // how many logs per second to generate
-	wg             *sync.WaitGroup     // notify when done
-	logger         *zap.Logger         // logger
-	index          int                 // worker index
-	traceID        string              // traceID string
-	spanID         string              // spanID string
+	running        *atomic.Bool    // pointer to shared flag that indicates it's time to stop the test
+	numLogs        int             // how many logs the worker has to generate (only when duration==0)
+	body           string          // the body of the log
+	severityNumber log.Severity    // the severityNumber of the log
```

Comment:
Is this the same number range as before?

Reply:
yup